requirements/all.txt

```diff
@@ -148,187 +125,69 @@ gevent==1.2.2 \
+hiredis==0.3.1 \
+    --hash=sha256:098d0c767115fb2ffee4f142d76ad86785829ef22b07a0ce2d289eef78186705 \
+    --hash=sha256:09f2d7bc980b2d1f402b39d4cc7841988493cf06580fd4b3120cbcf8a73402e5 \
+    --hash=sha256:1875ca8a843efd3220e1a39c0e754847030b7fd8ff4102c23d7ab7c791ccf0c2 \
+    --hash=sha256:2014924fb557541ee239e5035ddf8c4c03b166a0d6f45b265b1cad867091a617 \
+    --hash=sha256:22d814b567774c04459d7ff36979bacc4ada490d316579b3c357cfd949677c30 \
+    --hash=sha256:28c9da46e2f51d9a3b59dbf42e41278c979413bc0067a119fd51cf28a2f36379 \
+    --hash=sha256:4c0f4f3c94f4c9071e3f58546d05757a9e2e49e64769395963238548807ecd22 \
+    --hash=sha256:5ae5dd2d78ec1ea593207fa1c9bdc46da8b1713f038695f83457bc8307243ce2 \
+    --hash=sha256:602f667281cedf0e21c8648aaf72e70d6a0ccf7ee274b4bc9ea6aa889f27b66c \
+    --hash=sha256:66344a5bdf19b39ba3c00101663806ef92fd75b1451a2f3f33ca73352fabaa01 \
+    --hash=sha256:691e5c4efca2cf974ed24414af4d32c91bbf7650353504da918e0c29ad171d8b \
+    --hash=sha256:69a9d6c742ffd586093def8ac3dd1907907c8396b36af8b0490e9e52f0015e99 \
+    --hash=sha256:726e546d6b2133787b6c690176ed54d969f5c90f5bda650c5cbaa44bf18e4507 \
+    --hash=sha256:73227ed9d34648eae694cf3b4683840ea0e7dace682dcf3f1f732b42f9f0631c \
+    --hash=sha256:736a30992ede64b79989869e1a068fec93405dac71d87a316f22ee72e75cb6a5 \
+    --hash=sha256:79742f23dbb6a8dcf8b857ced49e81f872a792f587a98f0a4fe620a175eb8b70 \
+    --hash=sha256:7bf565198d612ad6b19a83b2d65a8e88d2b82608b0c4513c89ecfc25bd1dcce0 \
+    --hash=sha256:ae104a4a622320e0174fb82874d383b6b5fef0e3a38375695e01300107cc3e67 \
+    --hash=sha256:bdd78231aac38eadd00112c6eb8f6862ce9264a397602cd7eb6fe47e1c7273ba \
+    --hash=sha256:c6a7ab327dd01d77096f60bfaff1d0d5386960190fbd8a846ec8daafd529e0e3 \
+    --hash=sha256:cac4abfbd42d3bb7a363c51752f909645559617e3cbd2c9fd175159cd8bbc102 \
+    --hash=sha256:cf216d6fbacef0a20d23755d2d453dc6672ba5fd2b0cf76c548f4f620fce2cfe \
+    --hash=sha256:d8222d09da762891d4ef39374b1f2318da0ba02faf609d254438672892155eee \
+    --hash=sha256:dac079ff66c2d1c56f62f0a9b0a7a01784ad7b9eb136b9a931f7ab57463e70de \
+    --hash=sha256:e68638e460e95a68dbcef6f78be8210f4405eac020f10ee52cafee54084d3699 \
+    --hash=sha256:fb8b80d2fb45857fd19d04387e4207412124e470c3b5d4d3568a6d9b74a72e99 \
+    --hash=sha256:fc3eb53b51acf467ae7d60e92af23dee33b20871d84ce33e6e61b9e00077ff48 \
+    --hash=sha256:ffe6d9bc7f3637d962eb4706a2908ba4064ec997beb99540bb385a0f7d7fe39a
+
+# Messaging library
+# Code: https://github.com/celery/kombu
+kombu==3.0.37 \
+    --hash=sha256:7ceab743e3e974f3e5736082e8cc514c009e254e646d6167342e0e192aee81a6 \
+    --hash=sha256:e064a00c66b4d1058cd2b0523fb8d98c82c18450244177b6c0f7913016642650
+
+# Python MaxMind DB reader extension
+# Code: https://github.com/maxmind/MaxMind-DB-Reader-python
+maxminddb==1.3.0 \
+    --hash=sha256:972399b6d08b4cbb7889f2d2489266d18811c5ef6109cc52eb28de4102bbc4be
+
+# Mobile Country Codes (unmaintained)
+# Code: https://github.com/andymckay/mobile-codes
+mobile-codes==0.7 \
```

Comment:
[andymckay/mobile-codes](https://github.com/andymckay/mobile-codes) is marked as unmaintained.